concerned-coyotes/earlyinternet/earlyinternet/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path,include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('',include('weather.urls'))
 ]
```

Comment:
Weather currenlty has no views so you dont need to add its (nonexistent) urls

Reply:
I don't have this file in my editor